docs/controls/CameraPreview.md

```diff
@@ -0,0 +1,80 @@
+---
+title: CameraPreview
+author: skommireddi
+description: The CameraPreview control allows to easily preview video in the MediaPlayerElement from available camera frame source groups. You can subscribe and get real time video frames and software bitmaps as they arrive from the selected camera source. It shows only frame sources that support color video preview or video record streams.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, CameraPreview, Camera, Preview, Video Frame, Software Bitmap
+---
+
+# CameraPreview
+
+The **CameraPreview** control allows to easily preview video in the MediaPlayerElement from available camera frame source groups. You can subscribe and get real time video frames and software bitmaps as they arrive from the selected camera source. It shows only frame sources that support color video preview or video record streams.
+
+> [!IMPORTANT] Make sure you have the webcam capability enabled for your app to access the device's camera.
+
+## Syntax
+
+```xaml
+<controls:CameraPreview x:Name="CameraPreviewControl" 
+	FrameSourceGroupButtonIcon="ms-appx:///Assets/Photos/CameraSource.png"
+	FrameArrived="CameraPreviewControl_FrameArrived"
+	PreviewFailed="CameraPreviewControl_PreviewFailed">
+</controls:CameraPreview>       
+```
+
+```csharp
+private void CameraPreviewControl_FrameArrived(object sender, FrameEventArgs e)
+{
+	var videoFrame = e.VideoFrame;
+	var softwareBitmap = e.SoftwareBitmap;
+}
+
+private void CameraPreviewControl_PreviewFailed(object sender, FailedEventArgs e)
+{
+	var errorMessage = e.Error;
+}
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| FrameSourceGroups | IReadOnlyList<MediaFrameSourceGroup> | Gets a read only list of MediaFrameSourceGroups that support color video record or video preview streams. |
+| FrameSourceGroupButtonIcon | ImageSource | You can customize the icon for Frame Source Group button. |
+
+
+## Methods
+
+| Methods | Return Type | Description |
+| -- | -- | -- |
+| Dispose() | void | Use this method to dispose the control and media resources. |
+
+## Events
+
+| Events | Description |
+| -- | -- |
+| FrameArrived | Fires when a new frame arrives.|
+| PreviewFailed | Fires when camera preview fails.|
+
+```csharp
+private void CameraPreviewControl_PreviewFailed(object sender, FailedEventArgs e)
+{
+	var errorMessage = e.Error;
+}
+```
+
+## Sample Code
+
+[CameraPreview Sample Page](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CameraPreview)
+
+## Requirements (Windows 10 Device Family)
+
+| [Device family](http://go.microsoft.com/fwlink/p/?LinkID=526370) | Universal, 10.0.14393.0 or higher |
```

Comment:
Change this to match [documentation template](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/docs/.template.md)

```
## Requirements

| Device family | Universal, 10.0.14393.0 or higher   |
```